soh/soh/SohMenuBar.cpp

```diff
@@ -1337,6 +1339,22 @@ void DrawCheatsMenu() {
         UIWidgets::Tooltip("Prevents the Deku Shield from burning on contact with fire");
         UIWidgets::PaddedEnhancementCheckbox("Shield with Two-Handed Weapons", "gShieldTwoHanded", true, false);
         UIWidgets::Tooltip("This allows you to put up your shield with any two-handed weapon in hand except for Deku Sticks");
+        if (UIWidgets::PaddedEnhancementCheckbox("Hold Hylian Shield as Child Link", "gChildHoldsHylianShield", true, false)) {
```

Comment:
I'd say this belongs in the enhancements->gameplay->items menu more than the cheats menu

Reply:
I put it in cheats since it can be used to break logic, but I'm not opposed to moving it